scripts/kconfig/kconfigfunctions.py

```diff
@@ -91,8 +104,172 @@ def dt_str_val(kconf, _, name):
 
     return dt_defines[name].strip('"')
 
+
+def dt_chosen_to_label(kconf, _, chosen):
+    """
+    This function takes a 'chosen' property and treats that property as a path
+    to a EDT device.  If it finds a EDT device, it will look to see if that
+    device has a "label" property and return the value of that "label", if not
+    we return an empty string.
+    """
+
+    if doc_mode or edt is None:
+        return ""
+
+    dev = edt.chosen_dev(chosen)
+    if not dev:
+        return ""
+
+    if "label" not in dev.props:
+        return ""
+
+    return dev.props["label"].val
+
+def _dt_to_reg_addr(dev, index, unit):
```

Comment:
`_dev_reg_addr`?

Sounded like it mapped device trees to register addresses first.

Reply:
Changed.